tests/functional/object/mcg/ui/test_rgw_bucket_ui.py

```
@@ -0,0 +1,152 @@
+        if not obj_storage.is_rgw_provider_available():
+            pytest.skip("RGW provider card is not available or is disabled")
+
+        obj_storage.select_storage_provider(constants.S3_PROVIDER_RGW_INTERNAL)
+        bucket_ui.page_has_loaded(retries=15)
+
+        buckets = bucket_ui.get_buckets_list()
+        logger.info(f"RGW bucket list loaded with {len(buckets)} buckets")
+
+    @tier1
+    @pytest.mark.polarion_id("OCS-7743")
+    def test_create_rgw_bucket(self, setup_ui_class):
```

Ruff (0.15.21)

[warning] Unused method argument: `setup_ui_class`

(ARG002)